features/support/env.rb

```diff
@@ -83,3 +83,6 @@
 
 # Warden test helpers so the user authentication can be as fast as possible
 include Warden::Test::Helpers
+
+require_relative 'tables'
+World(TableUtils)
```

Comment:
It's the ["correct"](https://github.com/cucumber/cucumber/wiki/A-Whole-New-World) way to include functions in the global Cucumber context. I don't even know how the `include` line before it works!